File: math/FIzz_Buzz/fizzbuzz.py
#This program prints 'fizz' if a number within the given range is divisible by 3, 'buzz' if the number is divisible by 5 and 'fizzbuzz' the number is divisible by both 3&5.

min = int(input("Enter minimum for range : "))
max = int(input("Enter maximum for range : "))
for num in range(min,max+1):
    string = ""
    if num % 3 == 0:
        string = string + "Fizz"
    if num % 5 == 0:
        string = string + "Buzz"
    if num % 5 != 0 and num % 3 != 0:
        string = string + str(num)
    print(string)

File: math/FIzz_Buzz/javascript/fizzBuzz.js
// HI, My name is Adediwura Boluwatife


// This is a short program that prints each number from 1 to 100 on a new line 
//For each multiple of 3, print "Fizz" instead of the number 
//For each multiple of 5, print "Ghost" instead of the number 
//For numbers which are multiples of both 3 and 5, print "FIzzbuzz" instead of the number.

// Solution using a function,for loop and if statement

function fizzBuzz2(n) {
    for (let i = 1; i <= n; i++) {
      let str = "";
  
      if (i % 3 == 0) str += "fizz"
      if (i % 5 == 0) str += "buzz"
      if (i % 5 && i % 5 == 0) str += "fizzbuzz"
      if (str == "") str = i;
    
      console.log(str);
    }
  }